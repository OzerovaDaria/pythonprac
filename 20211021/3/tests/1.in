2
aa bb dd kk ss vvv gvg mxdskjnv;; cejdn kskkkkjjk 12jkjewkdmsmcl;;;;mmm


