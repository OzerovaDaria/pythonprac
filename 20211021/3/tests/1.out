aa bb dd kk ss 